sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebAppConfiguration.java

```diff
@@ -121,6 +136,21 @@ public static int resourceServerCount(Set<String> scopes) {
         return result;
     }
 
+    @ControllerAdvice
+    public class GlobalExceptionAdvice {
+        @ExceptionHandler(AADConditionalAccessException.class)
+        public void handleUserNotFound(HttpServletRequest request, HttpServletResponse response, Exception e) throws IOException {
+            if(e instanceof AADConditionalAccessException){
```

Comment:
Is this redundant because we already have `@ExceptionHandler(AADConditionalAccessException.class)`.